projects/bms_carrier/inc/bms_events.h

```diff
@@ -4,5 +4,17 @@ typedef enum {
   BMS_CAN_EVENT_RX = 0,
   BMS_CAN_EVENT_TX,
   BMS_CAN_EVENT_FAULT,
-  NUM_BMS_CAN_EVENTS
+  INVALID_BMS_CAN_EVENT
 } BmsCanEvent;
+#define NUM_BMS_CAN_EVENTS (INVALID_BMS_CAN_EVENT - BMS_CAN_EVENT_RX + 1)
```

Comment:
Like this pattern. We'll should probably use this pattern for other modules I think.

Reply:
Reason I went with this was because usage would be semi-similar to what it already is. Lowkey, I think just naming it `<event>_END` is sufficient to match the current pattern tho